### PR TITLE
feat: Button tailwind css classes

### PR DESCRIPTION
**Pull request by bladicito**

### CODE REVIEW

File: components/button/css.ts [hunk 1]
```diff
@@ -1,3 +1,21 @@
+const statesHover = [
```
Comment (lehmi11) [inline components/button/css.ts:1]: I like the way you structured the base styling of the button

File: components/icon/icon.css [hunk 2]
```diff
@@ -2,12 +2,15 @@
 
 @layer components {
   .c-icon {
-    display: inline-flex;
-    fill: currentColor;
+    display: inline-block;
```
Comment (lehmi11) [inline components/icon/icon.css]: is there a reason why you didn't use tailwind for the styling here? you can apply the tailwind css styles with the keyword @apply. 
e.g. 
```
@layer components {
  .btn-primary {
    @apply py-2 px-4 bg-blue-500 text-white font-semibold rounded-lg shadow-md hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-400 focus:ring-opacity-75;
  }
```

**Review by lehmi11 — APPROVED** [review]
Nice Job Bladimir. Great having such a versatile button.

File: components/icon/icon.css (hunk 2 above)
Comment (bladicito) [reply components/icon/icon.css]: I changed a lot of the Icon component in its own PR.

**Pull request merged**